### PR TITLE
*: upgrade mysql.user table schema

### PR DESCRIPTION
**Pull request by tiancaiamao**
After upgrade, mysql.user table schema is compatible to MySQL. New column add but not used yet.

### CODE REVIEW

File: bootstrap.go [hunk 1]
```diff
@@ -221,9 +250,15 @@ func upgrade(s Session) {
 	}
 	if ver < version3 {
 		upgradeToVer3(s)
+		ver = version3
```
Comment (zimulala) [inline bootstrap.go]: If add this line, the information is wrong.

File: mysql/const.go [hunk 2]
```diff
@@ -178,87 +178,180 @@ const (
...
+	CreateViewPriv:       "Create View",
+	ShowViewPriv:         "Show View",
+	CreateRoutinePriv:    "Create Routine",
+	AlterRoutinePriv:     "Alter Routine",
+	EventPriv:            "Event",
+	TriggerPriv:          "Trigger",
+	CreateTablespacePriv: "Create Tablespace",
+}
+
+// Priv2SetStr is the map for privilege to string.
```
Comment (zimulala) [inline mysql/const.go]: Update this comment.

**Comment by zimulala** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Pull request merged**